### PR TITLE
Add missing information for glib clone macro

### LINKED ISSUE #129: Update clone macro documentation to mention it needs glib in scope

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #129.

### CODE REVIEW

File: glib-macros/src/lib.rs [hunk 1]
```diff
@@ -25,6 +25,15 @@ use syn::{parse_macro_input, DeriveInput, LitStr};
 /// closure is immediately returning an optional default return value. If none is provided, `()` is
 /// returned.
 ///
+/// **/!\ IMPORTANT /!\\**
+///
+/// `glib` needs to be in scope, so unless it's one of your dependency, you need to import it
```
Comment (sdroege) [inline glib-macros/src/lib.rs:30]:
```suggestion
/// `glib` needs to be in scope, so unless it's one of the direct crate dependencies, you need to import it
```

File: glib-macros/src/lib.rs [hunk 2]
```diff
@@ -25,6 +25,15 @@ use syn::{parse_macro_input, DeriveInput, LitStr};
 /// closure is immediately returning an optional default return value. If none is provided, `()` is
 /// returned.
 ///
+/// **/!\ IMPORTANT /!\\**
```
Comment (sdroege) [inline glib-macros/src/lib.rs:28]:
```suggestion
/// **⚠️ IMPORTANT ⚠️ **
```

It's 2021!

**Review by sdroege — APPROVED** [review]

File: glib-macros/src/lib.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply glib-macros/src/lib.rs:28]: :o

**Comment by GuillaumeGomez** [comment]
Updated!

**Review by sdroege — APPROVED** [review]

**Pull request merged**